app/models/ability.rb

```diff
@@ -0,0 +1,13 @@
+class Ability
+  include CanCan::Ability
+
+  def initialize(user)
+    user ||= User.new
+    if user.admin?
+      can :manage, :all
+    else

```

Comment:
- [ ] Kindly consider adding another rule for the only owner can delete an item(food, recipe) if it is his own which has already been created